Demos/Cube/macOS/DemoViewController.m

```diff
@@ -45,15 +45,15 @@ -(void) viewWillAppear {
 	// Enabling this will sync the rendering loop with the natural display link
 	// (monitor refresh rate, typically 60 fps). Disabling this will allow the
 	// rendering loop to run flat out, limited only by the rendering speed.
-	_useDisplayLink = YES;
+	_useDisplayLink = NO;
 
 	// If this value is set to zero, the demo will render frames until the window is closed.
 	// If this value is not zero, it establishes a maximum number of frames that will be
 	// rendered, and once this count has been reached, the demo will stop rendering.
 	// Once rendering is finished, if _useDisplayLink is false, the demo will immediately
 	// clean up the Vulkan objects, or if _useDisplayLink is true, the demo will delay
 	// cleaning up Vulkan objects until the window is closed.
-	_maxFrameCount = 0;
+	_maxFrameCount = 2000;
```

Comment:
Did you mean to push these changes?

Reply:
Oh, for %#$ sake. 🤦🏻‍♂️

Thanks. Nice catch. Fixed now.